doc/user-guide.md

```diff
@@ -53,6 +53,7 @@ Each `RuntimeComponent` CR must at least specify the `applicationImage` paramete
 | `pullPolicy` | The policy used when pulling the image.  One of: `Always`, `Never`, and `IfNotPresent`. |
 | `pullSecret` | If using a registry that requires authentication, the name of the secret containing credentials. |
 | `initContainers` | The list of [Init Container](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.14/#container-v1-core) definitions. |
+| `extraContainers` | A YAML array of `containers` definitions. These are additional containers to be added to the pods. Note: the extraContainers can not be named `app`. |
```

Comment:
@leochr what do you think of naming the new field `sidecarContainers`?

Reply:
@navidsh +1, since "sidecar" terminology is widely used. Thanks.

Reply:
@JusteenR can you please rename `extraContainers ` → `sidecarContainers`?